packages/dds/merge-tree/src/test/beastTest.spec.ts

```diff
@@ -478,7 +472,7 @@ export function mergeTreeCheckedTest(): number {
 		0,
 		[TextSegment.make("the cat is on the mat")],
 		mergeTree.localPerspective,
-		{ seq: UniversalSequenceNumber, clientId: LocalClientId },
+		mergeTree.collabWindow.mintNextLocalOperationStamp(),
```

Comment:
note for this change and some other places like it: since the `MergeTree` hasn't had `startOrUpdateCollaboration` called, minting the next local operation stamp results in the same thing that happened previously (we don't generate local seq values and use `UnassignedSequenceNumber` before collaboration begins)